orm/internal/testpb/bank.proto

```diff
@@ -9,30 +9,30 @@ import "cosmos/orm/v1/orm.proto";
 message Balance {
   option (cosmos.orm.v1.table) = {
     id: 1;
-    primary_key: {
-      fields:
-          "address,denom"
-    }
-    index: {
-      id:
-          1 fields: "denom"
-    }
-  };
+primary_key: {
+fields:
+  "address,denom"
+}
+index: {
+id:
```

Comment:
Like here, it's some weird indent

Reply:
I don't like this either -- AFAIK, this is a config based on clang format, no?

Reply:
Yes, looking at the `.clang-format` configuration now